static/sass/_pattern_icon.scss

```diff
@@ -11,30 +11,30 @@
   .p-icon--oval-green {
     @extend %icon;
 
-    background-image: url("static/assets/img/status/status-succeeded.svg");
+    background-image: url("static/assets/img/status/status-running.svg");
```

Comment:
Do we need to include the custom icons? Can't we directly use the vanilla `p-icon--status-foo` icons? I think then we can remove the SVG's and this extra SCSS bit here.